### PR TITLE
add so.onekey.Wallet

### PR DESCRIPTION
**Pull request by proletarius101**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:** https://github.com/OneKeyHQ/app-monorepo/issues/2156
- [ ] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

File: so.onekey.Wallet.yml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+base-version: "22.08"
+command: onekey-wallet
+separate-locales: false
+finish-args:
+  - --device=all
+  - --share=ipc
+  - --share=network
+  - --socket=pulseaudio
+  - --socket=x11
+  - --socket=wayland
```
Comment (hfiguiere) [inline so.onekey.Wallet.yml:15]: it's either or. if it supprts wayland then it should be fallback-x11

File: so.onekey.Wallet.yml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+      - type: file
+        path: so.onekey.Wallet.metainfo.xml
+
+    build-commands:
+      - chmod +x OneKey-Wallet-linux.AppImage
+      - ./OneKey-Wallet-linux.AppImage --appimage-extract
+      - rm OneKey-Wallet-linux.AppImage
+
+      - desktop-file-edit --set-icon org.jitsi.jitsi-meet --set-key Exec --set-value
```
Comment (hfiguiere) [inline so.onekey.Wallet.yml:41]: really are you sure it's the right icon name?

File: README.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+# OneKey Wallet Flatpak
+
+Unofficial [Flathub package](https://flathub.org/apps/details/so.onekey.Wallet) of [OneKey Wallet](https://onekey.so/).
+
+You need to install **PCSC daemon** to use this app.
```
Comment (barthalion) [inline README.md:5]: Users are unlikely to ever see your README. You need to mention this requirement in the appdata file.

**Comment by barthalion** [comment]
As per https://github.com/OneKeyHQ/app-monorepo/issues/2156#issuecomment-1340266582, the appdata should also mention it's not officially supported – see [how Jetbrains app do it](https://flathub.org/apps/details/com.jetbrains.PhpStorm).

File: README.md (hunk 3 above)
Comment (barthalion) [reply README.md:5]: Up to you. Just pointing out that README is not user-facing.

**Pull request closed without merging**